File: excel/python/read/excel_read.py
#!/usr/bin/python
# -*- coding: utf-8 -*-
#
#	excel_read.py
#
#						Oct/16/2016
#
# -------------------------------------------------------------------
import sys
#
sys.path.append ('/var/www/data_base/common/python_common')
from excel_manipulate import excel_read_proc
from text_manipulate import dict_display_proc

# -------------------------------------------------------------------
sys.stderr.write ("*** 開始 ***\n")

excel_file = sys.argv[1]
print (excel_file)
#
try:
	dict_aa = excel_read_proc (excel_file)
	dict_display_proc	(dict_aa)
except Exception as ee:
	sys.stderr.write ("*** error *** in excel_read_proc ***\n")
	sys.stderr.write (str (ee))
#
#
sys.stderr.write ("*** 終了 ***\n")
# -------------------------------------------------------------------

File: excel/php/read/excel_read.php
#! /usr/bin/php
<?php
//
//	excel_read.php
//
//					Apr/14/2011
//
// ----------------------------------------------------------------
$path="/var/www/data_base/common/php_common";
set_include_path (get_include_path() . PATH_SEPARATOR . $path);

include "text_manipulate.php";
include "excel_manipulate.php";
// ----------------------------------------------------------------
print "*** 開始 ***\n";

$file_in = $argv[1];
#
print $file_in . "\n";

$dict_aa = excel_read_proc ($file_in);

dict_display_proc ($dict_aa);

echo " Peak memory usage: " . (memory_get_peak_usage(true) / 1024 / 1024) . " MB\r\n";

print "*** 終了 ***\n";

// ----------------------------------------------------------------
?>
